drivers/rtc/rtc-rv3028.c

```diff
@@ -722,6 +776,8 @@ static int rv3028_probe(struct i2c_client *client)
 					(void *)(rv3028->regmap),
 					RV3028_BACKUP, (void *)&backup, 1);
 			}
+			if (!ret)
```

Comment:
Is it necessary to refresh the RAM if the backup bits haven't changed?

Reply:
Nope. That should go inside the brace above. I'll fix.